dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -762,6 +766,17 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return getStrippedWorkflowsAndServices(userDAO.findById(user.getId()));
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @Path("/github/events")
+    @Operation(operationId = "getUserGitHubEvents", description = "Get all of the GitHub Events for the logged in user.", security = @SecurityRequirement(name = "bearer"))
+    @ApiOperation(value = "See OpenApi for details")
+    public List<LambdaEvent> getUserGitHubEvents(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user", in = ParameterIn.HEADER) @Auth User authUser) {
+        final User user = userDAO.findById(authUser.getId());
+        return lambdaEventDAO.findByUser(user);
```

Comment:
I realise this is just a draft, but this will quickly need pagination or something to cut things down. Plus sorting

Reply:
Do you know of a good endpoint that has pagination?

Reply:
Offset and limit control pagination here 
https://github.com/dockstore/dockstore/blob/develop/dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java#L176
Looks like its used by the old tools and workflows pages that paginate